aws/image/rhel/aws-rhel.pkr.hcl

```diff
@@ -0,0 +1,87 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 0.0.2"
+      source  = "github.com/hashicorp/amazon"
+    }
+  }
+}
+
+source "amazon-ebs" "rhel" {
+  ami_name      = "${var.ami_name}"
+  instance_type = "${var.instance_type}"
+  region        = "${var.region}"
+  source_ami_filter {
+    filters = {
+      name                = "RHEL-8.7.0_HVM-*"
```

Comment:
May the filter could also be a user variable so that RHEL9 could also use this script ?

Reply:
I think once will RHEL 9 will work let's just switch to it, the filter naming has very unique pattern sometimes (or we can have the two options)